packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -92,16 +93,18 @@ router.post(
   upload.single("file"),
   asyncHandler(async (req: Request, res: Response) => {
     const file = req.file;
-    if (!file) {
-      throw new BadRequestError("File must be provided");
-    }
-    const bundle = JSON.parse(file.buffer.toString());
+    if (!file) throw new BadRequestError("File must be provided");
+    const bundle = JSON.parse(file.buffer.toString()) as Bundle;
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const orgs = bundle.map((e: any) => e.resource.Organization);
+    const orgs: Organization[] = (bundle.entry as Organization[]) ?? [];
```

Comment:
the cast at the right already indicates the type, so no need to specify it on the left of the assignment